Added Release Notes for JBPM 7.44.0.Final version & BPMN Editor's new features (#2921)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.44.0.Final/inline-text-editing.adoc

[id='inline-text-editing-744']

> = Inline Text Editing

Author: Hey @handreyrc 
IMO this title is not clear about where this new feature applies to, so I just felt free to commit a fix for this title and specify it applies to both process & DMN editors.
Feel free to provide feedback also, let's try to wrap this up as soon as possible.
Thanks!

Reviewer: @romartin It's true! I agree.
Thanks.

> = Inline Text Editing in process & DMN designers

Reviewer 2: ```suggestion
= Inline Text Editing in process and DMN designer
```

> Node names editing has been made more fluid with the new _Inline Editor_.

Reviewer 2: ```suggestion

```

> By double clicking any node it is possible to edit text directly on the nodes without floating boxes or confirmation buttons.

Reviewer 2: ```suggestion

```

> For applying changes just hit _“Enter”_ or click outside of the node and for canceling hit _“Esc”_, while editing.

Reviewer 2: ```suggestion
You can double-click any node to edit its name directly without using floating boxes or confirmation buttons. After editing the node name in the inline text editor, press _Enter_ or click outside of the node to save the name. To cancel, press _Esc_ while editing. 
```
